web-api/src/personalization/routes.rs

```diff
@@ -195,6 +191,16 @@ struct PersonalizedDocumentData {
     properties: DocumentProperties,
 }
 
+impl From<PersonalizedDocument> for PersonalizedDocumentData {
+    fn from(value: PersonalizedDocument) -> Self {
+        Self {
+            id: value.id,
+            score: value.score,
+            properties: value.properties,
```

Comment:
:art: the argument name could be a bit less generic
```suggestion
    fn from(document: PersonalizedDocument) -> Self {
        Self {
            id: document.id,
            score: document.score,
            properties: document.properties,
```